🚧Update all contribution docs (#60)
KinsonDigital/.github: docs/CreatingBranches.md

<h2 style="border:0;font-weight:bold" align="center">Create a Branch to Contribute</h2>

When creating a branch, you need to determine if the feature or change you want to contribute is for a **production** release or a **preview** release.

If you want to release something to **production**, then you create your <span style="color: #66B2FF;font-weight:bold">feature</span> branch from the <span style="color: #82B366;font-weight:bold">main</span> branch. If you want to contribute to a **preview** release, you need to create your <span style="color: #66B2FF;font-weight:bold">feature</span> branch from the <span style="color: #CC99FF;font-weight:bold">preview</span> branch.

> **Note** Both branches will be merged into the same branch they were created from, via a pull request.

> Whatever method you want to use for creating your branches and pull requests is up to you and your preference.  But below is the git commands that you could use to help contribute to any of the **KinsonDigital** projects.

Reviewer: Grammar
```suggestion
Whatever method you want to use for creating your branches and pull requests is up to you and your preference.  But below are the git commands that you can use to help contribute to any of the **KinsonDigital** projects.
```

### **1. Sync Branch**

> As time moves on, the original repository changes.  This means that you need to keep your fork up to date as changes stream into the main repository.  It is important that you keep your fork up to date before you create a new branch.  Their are different options to do this.  To use the GitHub web interface go [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).  To use the GIT command line, follow the steps below.

Reviewer: Reword, contractions, grammar
```suggestion
As time moves on, the original repository will change which means it's important to keep your fork up to date before you create a new branch.  There are different options to do this.  To use the GitHub web interface go [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).  To use the GIT command line, follow the steps below:
```
